plugin/manager_linux.go

```diff
@@ -61,7 +61,7 @@ func (pm *Manager) enable(p *v2.Plugin, c *controller, force bool) error {
 	if err := pm.executor.Create(p.GetID(), *spec, stdout, stderr); err != nil {
 		if p.PluginObj.Config.PropagatedMount != "" {
 			if err := mount.Unmount(propRoot); err != nil {
-				logrus.Warnf("Could not unmount %s: %v", propRoot, err)
+				logrus.Warnf("Failed to %v", err)
```

Comment:
```suggestion
logrus.WithField("plugin", p.GetName()).WithError(err).Warn("Error unmounting plugin propagated mount root")
```

Even the old message was bad because it's not really giving any context about the problem.

Side note, we really shouldn't assume that the error message from the mount package is following some format since it's just a string that could change and would not realize that other messages should be updated.